README.md

```diff
@@ -29,6 +29,10 @@ tenonApi.checkUrl('http://www.example.com', function(err, result) {
 
 Each of the following methods takes an optional options object that can be
 More information about available options can be found in the [Tenon API documentation](https://bitbucket.org/tenon-io/tenon.io-documentation/src/master/src/2-understanding-api-request-parameters.md).
+### `analyze(url, [options,] callback)`
+
+Tests a given URL, HTML or HTML Fragment for accessibility issues and passes it through to checkUrl, checkSrc or checkFragment.
+
```

Comment:
This should probably reference that you can pass in any string, rather than a url, and the function will try to determine if it s a URL, HTML document or HTML fragment.